### PR TITLE
Fixes #133 UI avt in profile

### LINKED ISSUE #133: Profile - Avatar
Image

- Fix avatar is scaled 
- Text "Click to change image" is only appear when hover

### PR DESCRIPTION
**Pull request by lamanhvu62**

### CODE REVIEW

**Comment by lamanhvu62** [comment]
@lunguyenhat

File: src/components/Form/FormDAMImage/index.tsx [hunk 1]
```diff
@@ -26,6 +28,15 @@ const FormDAMImage = ({ current, onChoose }: any) => {
     setShow(false);
   };
 
+  useEffect(() => {
+    const prefersDark = window.matchMedia(
+      "(prefers-color-scheme: dark)"
+    ).matches;
+
+    console.log(prefersDark)
```
Comment (vietredweb) [inline src/components/Form/FormDAMImage/index.tsx:36]: Please remove debug here @lamanhvu62

**Review by vietredweb — CHANGES_REQUESTED** [review]
Please run below scripts before commiting code, to fix the CI @lamanhvu62 
- `yarn lint`
- `yarn format:check` (or you can just install Prettier - Code formatter on Visual Studio Code for automatic check)
- `yarn format:write`

**Comment by lamanhvu62** [comment]
i fix done @lunguyenhat  @vietredweb

**Review by vietredweb — APPROVED** [review]

**Review by vietredweb — CHANGES_REQUESTED** [review]
It seems the new commit missing `yarn format:check` and `yarn format:write` @lamanhvu62

**Review by vietredweb — APPROVED** [review]

**Review by vietredweb — APPROVED** [review]

**Pull request merged**